README.rst

```diff
@@ -194,7 +185,7 @@ packerlicious contains source code from `troposphere`_ which is licensed under t
 
 .. _`packer`: https://www.packer.io/
 .. _`troposphere`: https://github.com/cloudtools/troposphere
-.. _`LICENSE`: https://github.com/mayn/packerlicious/blob/master/LICENSE
+.. _`LICENSE`: https://github.com/dkasyanov/packerlicious/blob/master/LICENSE
```

Comment:
why change in license?

Reply:
sorry. I  was building project for myself and forgot about this PR.
reverted changes